droidlet/interpreter/craftassist/dance.py

```diff
@@ -59,11 +59,13 @@
 
 def add_default_dances(memory):
     """Add dances to agent's memory"""
-    memory.add_dance(generate_sequential_move_fn(jump), name="jump")
+    memory.add_dance(generate_sequential_move_fn(jump), name="jump", tags=["jump"])
```

Comment:
and another tag could be "dance" here, right ?

Reply:
I think not, we don't want a single jump to be returned when a player asks the agent to dance ?

Reply:
Jumps are dances though, right ?

Reply:
they are Dances in the sense of the Dance Task (a sequence of movements where the sequence is more important than the destination) but not in the standard common parlance.  I  am using this tag as a thing that can be searched over to return a dance when no particular dance is specified; but I think in that situation the person commanding the agent wants the agent to dance in the common sense